deploy/kube-templates/glusterfs-daemonset.yaml

```diff
@@ -58,6 +60,9 @@ spec:
         securityContext:
           capabilities: {}
           privileged: true
+        env:
```

Comment:
need to rebase this patch on master and merge the env section with existing one.